packages/e2e-tests/test/repl-helpers.ts

```diff
@@ -89,6 +89,7 @@ const fakeExternalEditor = async ({
       if (${JSON.stringify(flags ?? '')}) {
         assert.deepStrictEqual((${JSON.stringify(
           flags
+          // eslint-disable-next-line no-useless-escape
```

Comment:
Doesn't look like there is any useless escapes in the next line 🤔

Reply:
It doesn't recognise `\s` and thinks it is useless.

Reply:
Wow! That's pretty lame for a regex validating rule 😆